schemas/fields/urlField.ts

```diff
@@ -4,9 +4,18 @@ import ResourceUrlInput from "../../components/ResourceUrlInput";
 const urlField = defineField({
   title: "URL",
   description:
-    "This should be a valid web address, usually starting with 'https://'",
+    "This should be a valid web address, usually starting with 'https://'. Paths may be used but be aware this likely points to a specific resource, rather than a website overall.",
```

Comment:
Again, this is getting applied to the `urlField` - so don't forget to revert this change when moving the validation function to `website`. Feel free to 're-implement' in `website` - but you might need to change the language - don't think they'll know what a path is 😅

Reply:
Done